src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -238,7 +238,7 @@ function PopoverReportActionContextMenu(_props, ref) {
             Report.deleteReportComment(reportIDRef.current, reportActionRef.current);
         }
         setIsDeleteCommentConfirmModalVisible(false);
-    }, [reportActionRef]);
```

Comment:
Why are we making this change?

Reply:
`reportActionRef` is a variable that would never change (reportActionRef.current could change but that wouldn't cause rerender) - so I've changed it to line which is easier to read and has clearer intent. So it won't make developers think that callback is dependent on that variable.
From [react useRef docs](https://react.dev/reference/react/useRef)
> On the next renders, useRef will return the same object.